sync/command.py

```diff
@@ -268,10 +268,7 @@ def update_landing():
                                           allow_push=kwargs["push"],
                                           accept_failures=accept_failures)
         elif try_push.status == "complete" and not try_push.infra_fail:
-            landing.push_to_gecko(git_gecko,
-                                  git_wpt,
-                                  current_landing,
-                                  allow_push=kwargs["push"])
+            update_landing()
```

Comment:
Can we have a test for this change so that I can better understand which state this is covering?